libs/cua-driver-rs/scripts/install.ps1

```
@@ -0,0 +1,595 @@
+    [CuaDriverInstaller.Junction]::SetTarget($linkPath, $targetPath)
+}
+
+function Get-JunctionTarget([string]$linkPath) {
+    Add-JunctionSupportType
+    return [CuaDriverInstaller.Junction]::GetTarget($linkPath)
+}
+
+# Ensure a junction at $linkPath points at $targetPath. Refuses to clobber
+# an existing non-junction directory at $linkPath — the user may have
+# legitimate files there and we don't want to surprise them.
+function Ensure-Junction([string]$linkPath, [string]$targetPath) {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Ensure-Junction' uses an unapproved verb.

(PSUseApprovedVerbs)